PayloadPkg/OsLoader/OsLoader.c

```diff
@@ -1449,6 +1449,9 @@ PayloadMain (
   mEntryStack = Param;
 
   DEBUG ((DEBUG_INFO, "\n\n====================Os Loader====================\n\n"));
+  if (FeaturePcdGet (PcdEnableIppCryptoPerf))
+    IppCryptoPerf_Postmem();
+
```

Comment:
call it in Stage2 so the performance data is also available in UEFI payload?

Reply:
Allocating 10 MB data is not possible in stage2

Reply:
Update BoardConfig.py, When self.ENABLE_IPP_CRYPTO_PERF is set, increase self.LOADER_RSVD_MEM_SIZE.

Reply:
done

Reply:
since you added test in stage2, this change could be removed, right?

Reply:
I created a PR https://github.com/slimbootloader/slimbootloader/pull/2304 to update perf command. so we don't need update Osloader here.

Reply:
I have removed this call from Osloader.

Reply:
Osloader is still changed to print the test data. It is not necessary since perf command could do it.

Reply:
Is it ok add option to perf cmd to print only crypto related data?